.github/workflows/ci.yml

```
@@ -0,0 +1,98 @@
+          node-version: "18"
+          cache: "npm"
+          cache-dependency-path: "Cardoon/package-lock.json"
+
+      - name: 📦 Install dependencies
+        run: npm ci
+
+      - name: 🧪 Run tests
+        run: npm test -- --run --coverage
+
+      - name: 📊 Upload coverage reports
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.8)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)